tests/integration/test_handlers.py

```diff
@@ -1,3 +1,7 @@
+try:
+    from contextlib import asynccontextmanager  # Python 3.7
+except ImportError:
+    from async_generator import asynccontextmanager  # Python 3.6
```

Comment:
We'll drop 3.6 when 3.9 will be out I think. So, for now, we should be backward compat.